apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.ts

```diff
@@ -98,4 +113,28 @@ export class CellPopulationPredictionsComponent {
       this.router.navigate(['/us1']);
     }
   }
+
+  /** Triggered when clicked on download CSV button  */
+  onDownloadCSVButtonClicked() {
+    const fields = this.displayedColumns as (keyof CellSummaryRow)[];
+    const headers = Object.values(this.columnHeaders).join(',') + '\n';
+    const data = this.dataSource.data;
+    const rows = data
+      .map((row) =>
+        fields
+          .map((field) => {
+            if (field === 'percentage') {
```

Comment:
I don't think this is necessary. Downloading the data should give the raw data (not formatted)